src/UI/GameOption.java

```diff
@@ -6,27 +6,33 @@
 {
   public final String optionName;
   public final int minOption;
-  public final String[] optionList;
+  public final Object[] optionList;
   private int storedValue = 0;
 
-  public GameOption(String name, String[] Options, int defaultValue)
+  public GameOption(String name, Object[] Options, int defaultValue)
   {
     super(Options.length);
     minOption = 0;
     optionName = name;
     setSelectedOption(defaultValue);
     optionList = Options;
   }
-  public GameOption(String name, int min, int max, int defaultValue)
+  public GameOption(String name, int min, int max, int stride, int defaultValue)
   {
-    super(max - min);
-    minOption = min;
+    super(1 + (max - min)/stride);
```

Comment:
I think we really need to document that `(max - min)/stride` must be integer.
Things could get... wacky.

Reply:
It's all integer division, so it can't not be an integer. If you put in 0 to 50k with a stride of 235, you'll get 213 values out of it, exactly as you would expect.